dev/SSEConnection.js

```diff
@@ -104,7 +104,7 @@ function SSEConnection(connection, connectCallback) {
     var mPeer = connection.multiPeersHandler;
 
     function onMessagesCallback(message) {
-        if (message.remoteUserId != connection.userid) return;
+        if (message.remoteUserId != connection.userid && message.remoteUserId != conneciton.sessionid) return;
```

Comment:
hey @EtienneBruines is there a typo ?  should it be connection.sessionid

Reply:
Thank you very much!

I don't know how that one slipped through.

Fixed it now.